### PR TITLE
Update schema.json

### PR DESCRIPTION
**Pull request by laurensslats**
Editions, add fields:
- resellerURL
- video link
- email

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -73,6 +78,9 @@
           "type": "string",
           "format": "uri"
         },
+        "email": {
+          "type": "string"
+        },
```
Comment (johanstokking) [inline schema.json:83]:
```suggestion
        "email": {
          "type": "string",
          "format": "email"
        },
```

File: schema.json [hunk 2]
```diff
@@ -25,6 +25,11 @@
         { "pattern": "\\.(jpg|jpeg)$" }
       ]
     },
+    "video": {
+      "type": "string",
+      "format": "uri",
+      "pattern": "(https?:\/\/)www.(youtube.com\/watch[?]v=([a-zA-Z0-9_-]{11}))|(https?:\/\/)www.(youtu.be\/([a-zA-Z0-9_-]))|https?:\/\/(www.)?vimeo.com\/([0-9]{9})"
```
Comment (johanstokking) [inline schema.json:31]: Do you have a source for this? The video ID in the mobile YouTube URL seems off; no length check there.

File: schema.json [hunk 3]
```diff
@@ -930,6 +945,16 @@
...
+        "videos": {
+          "type": "object",
+          "properties": {
+            "main": { "$ref": "#/definitions/video" },
+            "other": {
+              "type": "array",
+              "items": { "$ref": "#/definitions/video" }
+            }
+          }
+        },
```
Comment (johanstokking) [inline schema.json:957]: Nice!

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: schema.json [hunk 4]
```diff
@@ -919,6 +927,13 @@
           "type": "string",
           "format": "uri"
         },
+        "resellerURL": {
```
Comment (johanstokking) [inline schema.json:930]:
```suggestion
        "resellerURLs": {
```

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: schema.json (hunk 4 above)
Comment (johanstokking) [reply schema.json:930]: Should we differentiate per region for the resellers?

Also with only URLs, we'll only be able to display a bullet list of URLs. If that's fine, then I'm good with it too.

File: schema.json (hunk 2 above)
Comment (laurensslats) [reply schema.json:31]: Updated. 

Played around with this tool: https://www.regextester.com/

It now supports all of these formats.
https://youtu.be/ZZbVsrYPOGE
https://www.youtube.com/watch?v=dsfasdf5623
http://www.youtube.com/watch?v=iwGFalTRHDA
http://www.youtube.com/watch?v=t-ZRX8984sc
http://vimeo.com/620922414
https://www.vimeo.com/462092214

File: schema.json (hunk 4 above)
Comment (laurensslats) [reply schema.json:930]: Yes, good point. Updated

File: schema.json [hunk 5]
```diff
@@ -25,6 +25,11 @@
         { "pattern": "\\.(jpg|jpeg)$" }
       ]
     },
+    "video": {
+      "type": "string",
+      "format": "uri",
+      "pattern": "^(https?:\/\/www.(youtube.com\/watch[?]v=([a-zA-Z0-9_-]{0,}))|(https?:\/\/)(www.)?(youtu.be\/([a-zA-Z0-9_-]{0,}))|https?:\/\/(www.)?vimeo.com\/([0-9]{0,}))"
```
Comment (johanstokking) [inline schema.json:31]:
```suggestion
      "pattern": "^(?:https?:\\\/\\\/(?:www\\.)?youtu(?:be\\.com\\\/watch\\?v=|\\.be\\\/)(?:[\\w\\-\\_]*)(?:&(amp;)?[\\w\\?=]*)?)$|^(?:https?:\\\/\\\/(?:www\\.)?vimeo\\.com\\\/(?:channels\\\/(?:\\w+\\\/)?|groups\\\/([^\\\/]*)\\\/videos\\\/|)(?:\\d+)(?:|\\\/\\?))$"
```

- More compact form for YouTube
- Allows for various Vimeos formats
- Full length match
- Fully escaped
- No accidental matches with unescaped `.`

File: schema.json [hunk 6]
```diff
@@ -919,6 +928,26 @@
...
+              "type": "object",
+              "properties": {
+                "name": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "region": {
+                    "type": "string",
+                    "minLength": 1
+                },
```
Comment (johanstokking) [inline schema.json:943]: Needs to be an array of enum

Let's go with the G20 excluding individual EU members. We can always expand later;

```suggestion
                "regions": {
                    "type": "array",
                    "items": {
                    	"enum": [
                    		"Argentina",
                    		"Australia",
                    		"Brazil",
                    		"Canada",
                    		"China",
                    		"European Union",
                    		"India",
                    		"Indonesia",
                    		"Japan",
                    		"Mexico",
                    		"Russia",
                    		"Saudi Arabia",
                    		"South Africa",
                    		"South Korea",
                    		"Turkey",
                    		"United States",
                    		"Other
                    	]
                    }
                },
```

**Review by johanstokking — CHANGES_REQUESTED** [review]

**Review by johanstokking — APPROVED** [review]

**Pull request merged**